docs/js/files/modules_FormModel.js.html

```diff
@@ -161,6 +161,12 @@ <h1 class="file-heading">File: modules/FormModel.js</h1>
     mapping: undefined,
 
     /**
+     * @property models
+     * @type Object
```

Comment:
can you add to this javadoc and the mapping javadoc to see the constructor for proper form/options for these fields?